base/ca/src/main/java/com/netscape/ca/CertificateAuthority.java

```diff
@@ -1844,10 +1845,15 @@ public SingleResponse processRequest(Request req) {
                 certStatus = new UnknownInfo();
             }
         } catch (EDBRecordNotFoundException e) {
-            // not found
-            certStatus = new GoodInfo(); // not issued not all
+            logger.debug(name + "cert record not found");
+            certStatus = new UnknownInfo(); // not issued by this CA
         } catch (EBaseException e) {
             // internal error
+            logger.debug(name + e.toString());
```

Comment:
Since this is an internal error, I'd suggest that we generate a proper error log to help troubleshooting, something like this:
```
logger.error("Unable to retrieve certificate: " + e.getMessage(), e);
```
This statement will show a user friendly error message for the admin and a stack trace for developers.